README.rst

```diff
@@ -5,7 +5,7 @@
 Waggle
 ======
 
-.. image:: https://img.shields.io/pypi/pyversions/waggle.svg
```

Comment:
What's the reason for this change?

Reply:
The shield was changed so that it shows Python 3.5, 3.6, 3.7.